File: cfgs/java_cfgs_foo/CHECK_IF_A_NUMBER_IS_JUMBLED_OR_NOT.java
Nodes
1;special;;;
2;conditional;(foo_1 / 10) == 0;14;
3;statement;return true;14;
4;exit;;;
5;conditional;foo_1 != 0;15;
6;conditional;(foo_1 / 10) == 0;16;
7;statement;return true;16;
8;statement;int digit1 = foo_1 % 10;17;
9;statement;int digit2 = (foo_1 / 10) % 10;18;
10;conditional;fun_1(foo_1 - foo_2) > 1;19;
11;statement;return false;19;
12;statement;num = foo_1 / 10;20;
13;statement;return true;22;
Edges
0;2;
1;3;5;
2;4;
3;
4;13;6;
5;7;8;
6;4;
7;9;
8;10;
9;11;12;
10;4;
11;5;
12;4;

File: cfgs/pythoncfgs/CHECK_IF_A_NUMBER_IS_JUMBLED_OR_NOT.py
Nodes
1;conditional;foo_1 // 10 == 0;8;
2;conditional;foo_1 != 0;10;
3;statement;return true;18;
4;conditional;foo_1 // 10 == 0;11;
5;statement;foo_1 = foo_2 % 10;13;
6;statement;foo_1 = foo_2 // 10 % 10;14;
7;conditional;fun_1(foo_1 - foo_2) > 1;15;
8;statement;foo_1 = foo_2 // 10;17;
9;statement;return false;16;
10;statement;return true;12;
11;statement;return true;9;
12;exit;;
Edges
0;2;11;
1;3;4;
2;12;
3;5;10;
4;6;
5;7;
6;8;9;
7;2;
8;12;
9;12;
10;12;
11;
